patches/netlify-lambda+1.6.3.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/node_modules/netlify-lambda/lib/build.js b/node_modules/netlify-lambda/lib/build.js
+index ac59043..6a07f3f 100644
+--- a/node_modules/netlify-lambda/lib/build.js
++++ b/node_modules/netlify-lambda/lib/build.js
+@@ -109,7 +109,7 @@ function webpackConfig(dir, { userWebpackConfig, useBabelrc } = {}) {
+     ],
+     output: {
+       path: functionsPath,
+-      filename: "[name].js",
++      filename: "[name].cjs",
```

Comment:
We can likely get away with getting rid of `netlify-lambda` entirely -- it's deprecated but the replacement, `netlify-cli`, ain't great. See #1052 

All we need is to compile the functions and, for local use, serve them.